lib/tasks/db.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+namespace :db do
```

Comment:
Nanobox doesn't support tasks that only run once on first deploy, unfortunately. I'm also not aware of a mechanism for determining whether the current deploy is the first (successful) one or not, other than the one given here which has Rake simply try to access the DB. I could try to approach this from a different angle that figures out whether this is an initial deploy or not some other way, but this is the cleanest approach since Rake tasks have access to the DB with the same settings as the rest of the application, without having to manually set them up externally.

The other alternatives I can think of, at the moment, are:

- make users do the initial DB setup manually (which is a bit involved, defeats some of the purpose of automated deploy tasks, and may actually work incorrectly given the automated deploy tasks would then run a `db:migrate` before the `db:setup` has a chance to be run)

- rename the task defined here so it doesn't sit inside an existing namespace (which would have no functional difference, but might be less scary to merge?)

Near as I can tell, though, this is the simplest and most effective option for running the `db:setup` task only when it's actually needed. So unless you really want me to change something, I'd feel safer leaving this as is.